src/Tools.EventStore/EventStoreClientBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="EventStoreClientBuilder.cs" company="Corsham Science">
+// Copyright (c) Corsham Science. All rights reserved.
+// </copyright>
+
+namespace CorshamScience.Tools.EventStore
+{
+    using System;
+    using global::EventStore.Client;
+
+    /// <summary>
+    /// A helper class for quickly building an <see cref="EventStoreClient"/>.
+    /// </summary>
+    public static class EventStoreClientBuilder
+    {
+        /// <summary>
+        /// Builds an <see cref="EventStoreClient"/> using the default settings.
+        /// </summary>
+        /// <param name="connectionString">Put valid connection string.</param>
+        /// <returns>EventStoreClient.</returns>
+        public static EventStoreClient GetEventStoreClient(string connectionString)
+        {
+            var settings = EventStoreClientSettings.Create(connectionString);
```

Comment:
Is this definitely all we need to specify compared to the old version? Seems like previously we were setting a gossipTimeout, is that not something we still need to support or is there a different way of setting it now?

Reply:
Discussed with @jbradford and @JamesMcCaig in Teams. Neither of us have been able to find references to this in the documentation, which suggests that the approach that we have taken here is sufficient